packages/cursorless-engine/src/processTargets/targets/ScopeTypeTarget.ts

```diff
@@ -145,6 +145,9 @@ export default class ScopeTypeTarget extends BaseTarget<ScopeTypeTargetParameter
   protected getCloneParameters() {
     return {
       ...this.state,
+      delimiter: this.insertionDelimiter,
+      removalRange: undefined,
+      interiorRange: undefined,
```

Comment:
I wasn't entirely sure what to do with these tbh. For constructing ranges (which is what "two items" does), these parameters are ignored anyway. And this code is equivalent to what we were doing before, just more explicit about it